status/status.go

```diff
@@ -38,6 +41,9 @@ func jsonResponse() string {
 		Success: true,
 		Info:    "Up and Running!",
 	}
-	bytes, _ := json.Marshal(response)
+	bytes, err := json.Marshal(response)
+	if err != nil {
+		panic(err)
```

Comment:
No panics from inside a web request. The status endpoint does not need to be json. We can make this just a success: true.

Reply:
Leave this as is and once this is merged, I'll push up a new PR with some changes to refactor this.